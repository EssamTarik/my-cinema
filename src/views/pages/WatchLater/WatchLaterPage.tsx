import React from "react";
import { WatchLaterMoviesContainer } from "../../../containers";

const WatchLaterPage = () => <WatchLaterMoviesContainer />;

export default WatchLaterPage;
