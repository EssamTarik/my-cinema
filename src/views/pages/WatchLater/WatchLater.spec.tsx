import React from "react";
import { render, cleanup } from "@testing-library/react";
import WatchLaterPage from "./WatchLaterPage";

jest.mock("../../../containers", () => ({
  WatchLaterMoviesContainer: () => <div />,
}));

describe("WatchLaterPage", () => {
  afterEach(cleanup);

  it("renders", () => {
    const { asFragment } = render(<WatchLaterPage />);
    expect(asFragment()).toMatchSnapshot();
  });
});
