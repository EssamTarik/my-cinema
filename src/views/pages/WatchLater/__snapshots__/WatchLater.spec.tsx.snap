// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WatchLaterPage renders 1`] = `
<DocumentFragment>
  <div />
</DocumentFragment>
`;
